src/components/blog/blogpostsDetails.js

```diff
@@ -2,6 +2,15 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.blogpostsDetails = void 0;
 exports.blogpostsDetails = [
+    {
+        "url": "https://blog.jupyter.org/gnu-octave-meets-jupyterlite-compute-anywhere-anytime-8b033afbbcdc",
```

Comment:
Just flagging since I have opened and saw. This url needs to be updated.

Reply:
Yes, thanks. I just did!